src/components/screens/wallet/send/form/formBtc.js

```diff
@@ -114,14 +120,14 @@ export default class FormBtc extends React.Component {
   }
 
   // TODO move dynamic fee calculation and presentation to a separate component
-  getCalculatedDynamicFee(dynamicFeePerByte, value) {
+  getCalculatedDynamicFee(dynamicFeePerByte) {
     const { fields: { amount }, unspentTransactionOutputs } = this.state;
     if (amount.error) {
       return 0;
     }
     const feeInSatoshis = btcTransactionsAPI.getTransactionFeeFromUnspentOutputs({
       unspentTransactionOutputs,
-      satoshiValue: toRawLsk(value || amount.value),
+      satoshiValue: toRawLsk(amount.value),
```

Comment:
💅🏻 Vit just a minor thing, I know you didn't change the whole function but maybe you can update it to something like this.

```suggestion
      getDynamicFeeCalculation(dynamicFeePerByte) {
    const { fields: { amount }, unspentTransactionOutputs } = this.state;
    return amount.error
    ? 0;
    : btcTransactionsAPI.getTransactionFeeFromUnspentOutputs({
      unspentTransactionOutputs,
      satoshiValue: toRawLsk(amount.value),
      dynamicFeePerByte,
    });
  }
```